src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -317,80 +333,140 @@ public void bindServiceInstance(String applicationName, String serviceInstanceNa
-                                                                                        .lifecycle(Lifecycle.builder()
-                                                                                                            .type(LifecycleType.BUILDPACK)
-                                                                                                            .data(BuildpackData.builder()
-                                                                                                                               .addAllBuildpacks(buildpacks)
-                                                                                                                               .build())
-                                                                                                            .build())
-                                                                                        .build())
+    private Lifecycle createDockerLifecycle() {
+        return Lifecycle.builder()
+                        .type(LifecycleType.DOCKER)
+                        .data(DockerData.builder()
+                                        .build())
+                        .build();
+    }
+
+    private ApplicationRelationships buildApplicationRelationships() {
+        return ApplicationRelationships.builder()
+                                       .space(buildToOneRelationship(getTargetSpaceGuid()))
+                                       .build();
+    }
+
+    private GetApplicationProcessResponse getApplicationProcess(UUID applicationGuid) {
+        return delegate.applicationsV3()
+                       .getProcess(GetApplicationProcessRequest.builder()
+                                                               .type("web")
+                                                               .applicationId(applicationGuid.toString())
+                                                               .build())
+                       .block();
+    }
+
+    private void updateApplicationProcess(Staging staging, GetApplicationProcessResponse applicationProcess) {
+        if (staging.isSshEnabled() != null) {
+            updateSsh(staging.isSshEnabled());
+        }
+        UpdateProcessRequest.Builder updateProcessRequestBuilder = UpdateProcessRequest.builder()
+                                                                                       .processId(applicationProcess.getId())
+                                                                                       .command(staging.getCommand());
+        if (staging.getHealthCheckType() != null) {
+            updateProcessRequestBuilder.healthCheck(buildHealthCheck(staging));
+        }
+        delegate.processes()
+                .update(updateProcessRequestBuilder.build())
+                .block();
+    }
+
+    private void updateSsh(boolean isSshEnabled) {
+        delegate.featureFlags()
```

Comment:
This is the wrong endpoint for enabling the application ssh feature.
The endpoint you're calling is this https://v3-apidocs.cloudfoundry.org/version/3.92.0/#feature-flags which doesn't support "ssh".
The endpoint we need to call is https://v3-apidocs.cloudfoundry.org/version/3.92.0/#update-an-app-feature 
This can be achieved with the `applicationsV3().updateFeature()` method

Reply:
Done, thanks